Twig/DescriptionExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceBundle\Twig;
+
+use Symfony\Cmf\Component\Resource\RepositoryRegistryInterface;
+use Puli\Repository\Api\ResourceRepository;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Puli\Repository\Api\Resource\PuliResource;
+use Symfony\Cmf\Component\Resource\Description\DescriptionFactory;
+
+/**
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+class DescriptionExtension extends \Twig_Extension
+{
+    private $descriptionFactory;
+
+    /**
+     */
+    public function __construct(
+        DescriptionFactory $descriptionFactory
+    ) {
+        $this->descriptionFactory = $descriptionFactory;
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('cmf_resource_description', array($this, 'getDescription')),
```

Comment:
or we could say that cmf_get_resource_description returns `false` if that information is not supported. or do you think `false` could be a desired information that we need to fetch and not do the same as if the information is not supported?